.gitignore

```diff
@@ -1,2 +1,3 @@
 .env
 node_modules
+**/.DS_Store
```

Comment:
I noticed that there is another pending PR that adds this to the gitignore as well so I can correct that conflict when we are ready to merge this PR.